src/get-transaction-path.coffee

```diff
@@ -0,0 +1,12 @@
+module.exports = (transaction) ->
+  origin = transaction['origin']
+  delimiter = ":"
```

Comment:
The only characters that are disallowed in identifiers `(`, `)`, `[` and `]` and new lines.